src/main/java/com/aws/greengrass/ipc/IPCEventStreamService.java

```diff
@@ -60,12 +61,17 @@ public class IPCEventStreamService implements Startable, Closeable {
     private SocketOptions socketOptions;
     private EventLoopGroup eventLoopGroup;
 
+    private static DeviceConfiguration deviceConfiguration;
+
     @Inject
     IPCEventStreamService(Kernel kernel,
                           GreengrassCoreIPCService greengrassCoreIPCService,
                           Configuration config,
                           AuthenticationHandler authenticationHandler) {
```

Comment:
Add `DeviceConfiguration` to the constructor. It will then be automatically injected. Do not manually lookup the DeviceConfiguration from the Context.